osh/lex.py

```diff
@@ -445,6 +445,15 @@ def IsKeyword(name):
   R(r'[^\\\0]+', Id.Char_Literals),
 ]
 
+# https://www.gnu.org/software/bash/manual/html_node/Controlling-the-Prompt.html#Controlling-the-Prompt
+PS1_DEF = _C_STRING_COMMON + [
```

Comment:
is this okay? you mentioned refactoring multiple groups into one common group

Reply:
Discussed on Zulip, we should not use _C_STRING_COMMON